### PR TITLE
mostly changed dataloader, but also updated fasttext

### PR DESCRIPTION
**Pull request by josegironn**

### CODE REVIEW

File: data_loader.py [hunk 1]
```diff
@@ -36,24 +37,37 @@ def tokenizer(text): # create a tokenizer function
     # Consider appending potentially named entity tags??
     return [tok.text for tok in spacy_en.tokenizer(text)]
 
+def genderToNum(gender):
+    assert(gender == 'm' or gender == 'f')
+    return 1 if gender == 'f' else 0
+
+def ageToInt(age):
+    return int(age[:-2])
```
Comment (N-Demir) [inline data_loader.py:45]: For the future I think you can just do like int(float(age)) possibly? I just ran into this too.

File: data_loader.py [hunk 2]
```diff
@@ -36,24 +37,37 @@ def tokenizer(text): # create a tokenizer function
...
+
 # Note that now everything is tsv but would like json!!
 def load_data(preprocessing=None):
     # Fields for the dataset
     # The actual review message
 
     #TEXT = Field(tokenize='spacy') # -- Old way, unclear exactly what language model is used
     TEXT = Field(sequential=True, tokenize=tokenizer, lower=True, preprocessing=preprocessing)
+    AGE = Field(sequential = False, use_vocab = False, dtype = torch.long, preprocessing = ageToInt)
+    GENDER = Field(sequential = False, use_vocab = False, preprocessing = genderToNum, dtype = torch.int)
```
Comment (N-Demir) [inline data_loader.py:55]: nit: Haha but the dtype and preprocessing are in reverse order :P

File: preprocess.py [hunk 3]
```diff
@@ -167,7 +167,7 @@
 		X.append(line.split(DELIMITER)[0:3])
 		Y.append(line.split(DELIMITER)[3])
 
-X_train, X_valid, Y_train, Y_valid = train_test_split(X, Y, test_size=0.2)
+X_train, X_valid, Y_train, Y_valid = train_test_split(X, Y, test_size=0.2, random_state=315)
```
Comment (N-Demir) [inline preprocess.py:170]: What kind of a savage uses the date for the random_state.

Comment (N-Demir) [reply preprocess.py:170]: 😄

**Review by N-Demir — APPROVED** [review]
Dope!

File: preprocess.py (hunk 3 above)
Comment (josegironn) [reply preprocess.py:170]: big fucking ballers that's who

**Pull request merged**